omod-1.9/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_9/ObsController1_9Test.java

```diff
@@ -84,6 +87,40 @@ public void shouldGetAll() throws Exception {
 	}
 	
 	@Test
+	public void searchByEncounter_shouldGetVoidedObsIfIncludeAllIsTrue() throws Exception {
+		
+		executeDataSet("encounterWithObsGroup1_9.xml");
+		
+		String encounterUuid = "62967e68-96bb-11e0-8d6b-9b9415a91465";
```

Comment:
Have you looked at our naming conventions in regards to the above line? https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-NamingConventions

Reply:
@dkayiwa 
This has been fixed.

Reply:
How have you fixed it?